pkg/cmd/util/helpers.go

```diff
@@ -149,6 +151,34 @@ func RetrieveApplicationsByName(ctx context.Context, c client.Client, applicatio
 	return applicationMetaList, nil
 }
 
+func RetrieveApplicationStatusByName(ctx context.Context, c client.Client, applicationName string, namespace string) ApplicationStatusMeta {
+	var applicationStatusMeta ApplicationStatusMeta
+	var appConfig corev1alpha2.ApplicationConfiguration
+	if err := c.Get(ctx, client.ObjectKey{Name: applicationName, Namespace: namespace}, &appConfig); err == nil {
```

Comment:
We usually check `err!=nil` first. Then all these logic could be written out of this brace

Reply:
Done. Instead I have to write `return` several times, but as you said, the logic seems more clear.